examples/custom_object.py

```diff
@@ -12,37 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This example use an example CRD from this tutorial:
-# https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/
-#
-# The following yaml manifest has to be applied first:
-#
-# apiVersion: apiextensions.k8s.io/v1beta1
-# kind: CustomResourceDefinition
-# metadata:
-#   name: crontabs.stable.example.com
-# spec:
-#   group: stable.example.com
-#   versions:
-#     - name: v1
-#       served: true
-#       storage: true
-#   scope: Namespaced
-#   names:
-#     plural: crontabs
-#     singular: crontab
-#     kind: CronTab
-#     shortNames:
-#     - ct
+"""
+Uses a Custom Resource Definition (CRD) to create a custom object, in this case
+a CronTab. This example use an example CRD from this tutorial:
+https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/
 
+The following yaml manifest has to be applied first:
 
-from pprint import pprint
+cat <<EOF | kubectl delete -f - 
```

Comment:
This is not part of the YAML manifest, but a bash script.

Reply:
Good catch, that should be 'create', not 'delete', I will update that. Either way, I'm just providing the user an easy way to apply the YAML in one easy copy-and-paste command since it's required before running the Python script. Let me know if I'm missing something.